CONTRIBUTORS.txt

```diff
@@ -111,28 +127,35 @@ needs to be listed here.
 - Rémi Doreau <remi.d45@gmail.com>
 - Saleem Abdulrasool <compnerd@compnerd.org>
 - Sebastian Thiebaud <770074+SebastianThiebaud@users.noreply.github.com>
+- Sebastian Vogt <sebastian1210@web.de>
 - Semen Zhydenko <simeon.zhidenko@gmail.com>
 - Sergej Jaskiewicz <jaskiewiczs@icloud.com>
 - Sergo Beruashvili <beruashvili@gmail.com>
 - Shaun Hubbard <shaunhubbard2013@icloud.com>
 - Shekhar Rajak <shekhar_rajak@apple.com>
+- Si Beaumont <beaumont@apple.com>
 - Siemen Sikkema <s.h.sikkema@gmail.com>
 - Simon Evans <si@si.org>
+- Stepan Ulyanin <99296376+stepan-ulyanin@users.noreply.github.com>
 - Stuart Hinson <stuart.hinson@gmail.com>
 - Tanner <me@tanner.xyz>
 - Tiago Martinho <martinho@thoughtworks.com>
 - Tibor Bödecs <mail.tib@gmail.com>
+- Tim Condon <0xTim@users.noreply.github.com>
 - Tim Condon <0xtimc@gmail.com>
 - Tim Kientzle <tkientzle@apple.com>
 - Timothée Peignier <timothee.peignier@icloud.com>
 - Tom Counsell <tamc@greenonblack.com>
 - Tomer Doron <tomerd@apple.com>
+- Tony Arnold <tony@thecocoabots.com>
 - Trevör <adtrevor@users.noreply.github.com>
 - Trevör Anne Denise <adtrevor@users.noreply.github.com>
 - Volodymyr Sapsai <vsapsai@users.noreply.github.com>
 - Will Lisac <will@lisac.org>
 - Wilson Ding <hello@wilsonding.com>
+- akash-55 <61596874+akash-55@users.noreply.github.com>
 - buttaface <butta@fastem.com>
+- buttaface <repo@butta.fastem.com>
```

Comment:
@buttaface which one of the two is your preferred email address for the contributors list?

Reply:
(also if you would like, we can add a full real name here if you let me know what you'd like it to read)

Reply:
The first is the real email address, the second is an alias I use on github in case I get spam, so I can throw it away later. If this list isn't getting spammed, the first one is fine. I don't use a real name, my nick is fine.